### ISSUE TITLE
[docs] redoc options

### ISSUE THREAD

**Issue by floggle** [Redocly/redoc#763]
Maybe I'm just very dense, but from the docs I can't figure out how to use the redoc options.

I'm using the redoc.standalone.js file, and the HTML call:
```
<redoc spec-url='http://example.com/openapi.json'></redoc>
```

So I want to try my hand at the theme stuff. It's not documented, fine, but the "options" business is. So I try:
```
<redoc spec-url='http://example.com/openapi.json' theme="./theme.ts"></redoc>
```
I get an error:

           e.spec.info is undefined

Ok, so lets try something else:
```
<redoc spec-url='http://example.com/openapi.json' disableSearch="True"></redoc>
```
Same error.
Ok, so maybe that business about "kebab"'s and tags (it's not really clear what "standalone on tag" means btw):
```
<redoc spec-url='http://example.com/openapi.json'  disable-search="True"></redoc>
```
Nope

Hmm, what this:
```
<redoc spec-url='http://example.com/openapi.json'  options="disableSearch"></redoc>
```
or
```
<redoc spec-url='http://example.com/openapi.json'  options="disable-search"></redoc>
```

Nope and nope.

I've tried googling around but can't find any examples of this being used. Your own example page uses javascript instantiation mixed in with GoogleAnalytics code (which is really bad from a "learn by example" thing - there should be no foreign code in there, or at least if there must be, it should be obviously segregated).

Short version - I have absolutely no idea how to trigger the options based on the docs.

**Comment by RomanGotsiy** [comment]
Try validating your spec. Looks like you don't have `info` section which is required.

Options don't change anything.

**Comment by christopher-bonitz** [comment]
> 
> 
> Try this:
> 
> ```
> <redoc spec-url='http://example.com/openapi.json' disable-search></redoc>
> ```

That right there, should be added to the main readme page, it is really useful to have full usage examples.
Annoying as it may be this thing took me a few hours to figure out, we're all working in different software layers, assumptions should be avoided.

**Comment by abdurrahman** [comment]
Maybe its time to update way of [HTML implementation](https://redocly.com/docs/redoc/deployment/html/) instead of giving this difficulty.

Besides of that, I guess there are some limitations in the options, disabling sidebar is not working for instance. 😕

**Comment by RomanHotsiy** [comment]
Only those options that are marked with "Supported in Redoc CE." are working with Redoc open source.

**Comment by RomanHotsiy** [comment]
I'll close this issue, I believe it's not actionable.
